### DISCUSSION TITLE
How to automatically remove the historical logs after a period of time?

### DISCUSSION THREAD

**Discussion by teklot** [odygrd/quill#294] [Q&A]
Is there any flag to have the quill remove the historical logs after a period of time?

**Comment by odygrd** [comment]
hey, the library not edit log files to remove lines from them.

The rotating file handlers offer a backup_count for the rotating files and they are removing older files, but that backup count resets when you restart the application

Another solution would be to clean up the old files based on their creation time with a script (bash/python) that you can run each time you start your application or on crontab
